backend/api/Database/Models/Inspection.cs

```diff
@@ -46,8 +46,7 @@ or InspectionStatus.Successful
 
         public float? VideoDuration { get; set; }
 
-        [MaxLength(250)]
-        public string? AnalysisTypes { get; set; }
+        public AnalysisType AnalysisTypes { get; set; }
```

Comment:
AnalasysType should singular (without the s) or a list

Reply:
Done.